chapters/arrays.tex

```diff
@@ -1010,7 +1010,7 @@ \section{Indexing}\label{array-indexing}\label{indexing}
 \end{center}
 \end{table}
 
-\subsection{Indexing with Boolean or Enumeration Values}\label{indexing-with-boolean-or-enumeration-values}
+\subsection{Boolean or Enumeration Indices}\label{boolean-or-enumeration-indices}
```

Comment:
Ok, although it loses the similarity to "Indexing with end"

Reply:
('end Index' looks ugly. The best I could think of was 'Using end Index', but it is not so similar.)

Reply:
To me it didn't look too problematic that the similarity was lost.  The fact that the new section names are concise makes them look good anyway.